src/core.ts

```diff
@@ -235,6 +235,11 @@ export type AirgapConsentEventType =
  */
 export type AirgapEventType = AirgapConsentEventType | 'purpose-map-load';
 
+export interface ConsentChangeEventPayload {
```

Comment:
Looks like I got it backwards!

Reply:
@csmccarthy is this a real API change? or did we get the type wrong the first time around

Reply:
I assume the latter, but if the former, then I don't recommend changing any public APIs.. this would break stuff for customers who are listening to this event

Reply:
@anotherminh Definitely the latter, I just forgot we had the consent values nested one field down

Reply:
awesome! this LGTM then!